javascript/ql/test/experimental/Security/CWE-073/ExpressHbsLFR.ql

```diff
@@ -0,0 +1,44 @@
+/**
+ * @name Express-Hbs Local File Read and Potential RCE
+ * @description Writing user input directly to res.render of ExpressJS used with Hbs can lead to LFR
```

Comment:
The name normally describes what happens, and not which kind of exploits that can lead to.   
I've given another suggestion for the name and description. 

```suggestion
 * @name Template object injection
 * @description Instantiating a template using a user-controlled object is vulnerable to local file read or remote code execution. 
```

Feel free to use the suggestion, or ignore if it you think you got a better name. 

(If you change the name, then the file name and `@id` should also be changed).

Reply:
Changed the Name to "Template Object Injection"

Reply:
Thanks for the appropriate suggestion.